editor/plugins/script_editor_plugin.cpp

```diff
@@ -2033,7 +2044,82 @@ Error ScriptEditor::_save_text_file(Ref<TextFile> p_text_file, const String &p_p
 	return OK;
 }
 
+bool ScriptEditor::open_script_in_external_editor(const RES &p_resource, int p_line, int p_col) {
+
+	if (!p_resource->get_path().is_resource_file()) {
```

Comment:
Empty lines after the opening brace are no longer allowed in the latest master. `clang-format` should be able to fix this for you.